modules/component-store/src/component-store.ts

```diff
@@ -125,15 +122,11 @@ export class ComponentStore<T extends object> implements OnDestroy {
         : of(observableOrValue);
       const subscription = observable$
         .pipe(
-          concatMap((value) =>
-            this.isInitialized
-              ? // Push the value into queueScheduler
-                scheduled([value], queueScheduler).pipe(
-                  withLatestFrom(this.stateSubject$)
-                )
-              : // If state was not initialized, we'll throw an error.
-                throwError(() => new Error(this.notInitializedErrorMessage))
-          ),
+          // Push the value into queueScheduler
+          observeOn(queueScheduler),
```

Comment:
I was very confident that `observeOn` was deprecated in favour of `scheduled`. Apparently I was dreaming?

Reply:
😁 AFAIK, the signature of the `on` function that accepts scheduler is deprecated in favor of `scheduled`.